.gitignore

```diff
@@ -7,5 +7,6 @@ cmd/VERSION
 .idea
 .vscode
 /pebble
+__debug_bin*
```

Comment:
Out of interest, what puts these files here?

Reply:
Ah, I had assumed it was just Go when it panicked or something like that. Looking into it, I think it's [delve](https://github.com/go-delve/delve), which VS Code had me install when I did some debugging in the IDE (for what turned out to be a typo...).

Since it's tool specific like that I'll remove it from here.